packages/devtools_app/lib/src/shared/analytics/constants.dart

```diff
@@ -76,10 +76,8 @@ const disableOpacityLayersOptionDocs = 'disableOpacityLayersDocs';
 const disablePhysicalShapeLayersOption = 'disablePhysicalShapeLayers';
 const disablePhysicalShapeLayersOptionDocs = 'disablePhysicalShapeLayersDocs';
 const canvasSaveLayerDocs = 'canvasSaveLayerDocs';
-const intrinsicOperationsDocs = 'intrinsicOperationsDocs';
```

Comment:
this is a bug that this is unused. Could you please replace `shaderCompilationDocs` with `intrinsicOperationsDocs` here instead of removing? https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/performance/panes/frame_analysis/frame_hints.dart#L269